zcutil/build.sh

```diff
@@ -33,14 +33,6 @@ if [[ -z "${HOST-}" ]]; then
     HOST="$BUILD"
 fi
 
-# Allow override to $CC and $CXX for porters. Most users will not need it.
-if [[ -z "${CC-}" ]]; then
-    CC=gcc
-fi
-if [[ -z "${CXX-}" ]]; then
-    CXX=g++
-fi
```

Comment:
Hmm. What's the rationale for removing this?

Reply:
See the corresponding commit message:

> zcutil/build.sh: Use config.site to set default ./configure settings
>
> This is a more reliable way to configure the depends path (matching what is done
> upstream), and frees up --prefix for user configuration.

`config.site` gets filled with the correct CC and CXX values during compilation of dependencies.